source/_docs/mqtt/broker.markdown

```diff
@@ -92,6 +92,31 @@ mqtt:
   certificate: /home/paulus/downloads/mosquitto.org.crt
 ```
 
+### HiveMQ Cloud
+
+HiveMQ Cloud is a fully managed MQTT broker that provides you a private broker. A free plan for up to 100 devices is available.
+
+Home Assistant is not affiliated with HiveMQ Cloud nor will receive any kickbacks.
+
+1. [Create an account](http://console.hivemq.cloud) <links to signup>
+2. With your signup you will receive automatically the free plan that allows you to connect up to 100 devices
+3. Create MQTT credentials in the Access Management tab of your Cluster Detail View you can use to connect Home Assistant and any MQTT device.
+   ![image2](../../images/mqtt/image2.png)
+4. Download the trusted certificate from let’s encrypt to ensure secure communication between Home Assistant and your HiveMQ Cloud cluster. <link: https://letsencrypt.org/certs/trustid-x3-root.pem >
```

Comment:
Please do not use bare links.

For my own learning: Do we really need to add the default trusted cert for Let's Encrypt? That thing is already available on the systems, right? Doesn't it pick it up?

Reply:
Yes, I think we do need this certificate, as HiveMQ Cloud uses TLS.

Reply:
At least in our tests it didn't work without this certificate. We tested on with Docker on Mac.